TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/PATCH-Negative-Basicinfo.robot

```diff
@@ -0,0 +1,29 @@
+*** Settings ***
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource     TAF/testCaseModules/keywords/core-metadata/coreMetadataAPI.robot
+Suite Setup  Run Keywords  Setup Suite
+...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Teardown Keywords
+Force Tags      v2-api
+
+*** Variables ***
+${SUITE}          Core Metadata Device Profile PATCH Basicinfo Negative Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-deviceprofile-patch-basicinfo-negative.log
+
+*** Test Cases ***
+ProfilePATCH001 - Update basicinfo with name validation error
```

Comment:
test case id is duplicated with the positive cases
use something like `ErrProfileBasicInfoPATCH001`